Sources/Risk/Services/RiskError.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  RiskError.swift
+//
+//
+//  Created by Precious Ossai on 13/10/2023.
+//
+
+import Foundation
+
+public enum RiskError: LocalizedError, Equatable {
```

Comment:
I believe we can remove the `LocalizedError` conformance here. Maybe even also `Equatable`